app/src/main/AndroidManifest.xml

```diff
@@ -20,6 +20,7 @@
         android:icon="@drawable/ic_launcher"
         android:label="@string/app_name"
         android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true"
```

Comment:
bad

Reply:
@iamareebjamal as stated in the comment, I was waiting for the pr #2263 to get merged, so to temporary make it work, I added that line. I would remove it, as #2263 is now merged. : )

Reply:
Removed it